hudi-hadoop-mr/src/main/java/org/apache/hudi/hadoop/utils/shims/Hive3Shim.java

```diff
@@ -40,7 +40,8 @@ public class Hive3Shim implements HiveShim {
   public static final String TIMESTAMP_WRITEABLE_V2_CLASS = "org.apache.hadoop.hive.serde2.io.TimestampWritableV2";
   public static final String DATE_WRITEABLE_V2_CLASS = "org.apache.hadoop.hive.serde2.io.DateWritableV2";
 
-  private static Class<?> TIMESTAMP_CLASS = null;
+  private static Class<?> TIMESTAMP_CLZZ = null;
```

Comment:
CLZZ -> CLASS ?

Reply:
I was trying to standardise the usage of `CLASS` = string.

`CLZZ` = the Class instant
This is so as we will need to create a `TimestampWritableV2` Class variable a few lines down too.

Since `TIMESTAMP_WRITEABLE_V2_CLASS` was used to define the string already, i had to change it to `TIMESTAMP_WRITEABLE_V2_CLZZ` define its class. 

Hence, the changes for the rest of the references to standardize the code style.

Reply:
> CLZZ = the Class instant

Sorry, is this a commonsense abbreviation?

Reply:
https://stackoverflow.com/questions/2529974/why-do-java-programmers-like-to-name-a-variable-clazz

Reply:
yeah, CLAZZ is what I like to use, but what is CLZZ?